app/src/main/java/org/jellyfin/androidtv/model/itemtypes/Episode.kt

```diff
@@ -0,0 +1,14 @@
+package org.jellyfin.androidtv.model.itemtypes
+
+import android.graphics.Bitmap
+
+data class Episode (
+	val id: String,
+	val canDelete: Boolean,
```

Comment:
The properties for "canDelete" and "canDownload" are user-permissions and can't be set per-item or per-library afaik. So I'd remove them from the episode pojo and move them somewhere else

Not sure if you can find this information in the UserDto though

Reply:
Ah, then I'll remove them. Supporting this is probably also pointless for a TV experience (at least at first)